homeassistant/components/cloud/__init__.py

```diff
@@ -308,12 +308,22 @@ async def _on_initialized():
 
     account_link.async_setup(hass)
 
-    async_call_later(
+    _cancel_repairs_callback = async_call_later(
```

Comment:
I dont know if it matters, but this task and on stop registration will remain allocated during the whole life of ha.

Maybe we should have a wrapper around call_later that also deregisters the event listener when the task have finished?

Reply:
I've added a new argument to `async_call_later` that will deregister the event listener.

Reply:
It will also be useful for #90834